docs/quickstart.md

```diff
@@ -13,32 +13,463 @@ You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/r
 
 ### Deploying Sedna
 
-Please refer to this [link](setup/install.md).
+Sedna provides two deployment methods, which can be selected according to your actual situation:
 
-### Prepare
+- Install Sedna on a cluster Step By Step: [guide here](setup/install.md).
+- Install Sedna AllinOne : [guide here](setup/local-up.md).
 
-todo
+### Component
+Sedna consists of the following components：
 
-### Configure
+![Architecture](./proposals/images/framework.png)
 
-todo
+#### GlobalManager
+* Unified edge-cloud synergy AI task management
+* Cross edge-cloud synergy management and collaboration
+* Central Configuration Management
 
-### Run
+#### LocalController
+* Local process control of edge-cloud synergy AI tasks
+* Local general management: model, dataset, and status synchronization
 
-todo
 
-### Monitor
+#### Worker
+* Do inference or training, based on existing ML framework.
+* Launch on demand, imagine they are docker containers.
+* Different workers for different features.
+* Could run on edge or cloud.
 
-todo
+
+#### Lib
+* Expose the Edge AI features to applications, i.e. training or inference programs.
+
+
+### System Design
+
+There are three stages in a [incremental learning job](./proposals/incremental-learning.md): train/eval/deploy.
+
+![](./proposals/images/incremental-learning-state-machine.png)
+
+## Deployment Guide
+
+### 1. Prepare
+
+#### 1.1 Deployment Planning
```

Comment:
The Deployment section is duplicate with https://github.com/kubeedge/sedna/blob/main/docs/setup/install.md
Better to let user just refer it.